parameter/ParameterMgrFullConnector.cpp

```diff
@@ -40,6 +42,11 @@ CParameterMgrFullConnector::CParameterMgrFullConnector(const string &strConfigur
 {
 }
 
+CommandHandlerInterface *CParameterMgrFullConnector::createCommandHandler()
+{
+    return new CommandHandlerWrapper(_pParameterMgr->createCommandHandler());
```

Comment:
This new is not useful  CommandHandlerInterface should be returned by copy. Actually can a return by copy not be used everywere (since with move semantic it is cheap) ?